data/json/items/tools.json

```diff
@@ -6120,7 +6128,7 @@
     "description": "This is a sealed glass jar containing V8.  Use to open and drink to enjoy.",
     "price": 120,
     "material": ["glass", "veggy"],
-    "weight": 522,
+    "weight": 500,
```

Comment:
I should double check the weight on this one,  I might have missed the weight on the V8